NextmediaMa/youcan-payment-php-sdk
# DOT-4698: Check YouCan Pay keys on save gateway

## JIRA Ticket

[DOT-4698](https://nextmediama.atlassian.net/browse/DOT-4698)

## QA Steps

```php
// check keys are valid
YouCanPay::instance()->keys->check('private-key', 'public-key');
// or using current instance
$youCanPay->keys->check('private-key', 'public-key');
```
